docs/tools/terraform/reference/cookbook/kafka-debezium-postgres-source.rst

```diff
@@ -0,0 +1,197 @@
+service with a Debezium source connector for PostgreSQL enabled and configured to connect to the PostgreSQL database and capture any change in tables.
+As soon as any of the monitored tables is inserted or updated with new data, the Debezium connector will capture the data change and convert table data into
+json payload and produce messages to the relevant Kafka topic. 
+Some of these services are created in one cloud provider and some in another cloud provider, to demonstrate how easy it is with Aiven to integrate services across 
+multiple cloud vendors.
+
+.. Warning::
+
+    Creating a `dedicated Aiven for Apache Kafka Connect service <https://developer.aiven.io/docs/products/kafka/kafka-connect/getting-started.html#apache-kafka-connect-dedicated-cluster>`_ is Aiven's suggested option. Having Kafka Connect running on the same nodes as Apache Kafka increases the load on the nodes possibly making the cluster more unstable. 
+    
+    For a better separation of concerns, a dedicated Aiven for Apache Kafka Connect cluster is therefore suggested.
+
+
+.. Tip::
+
+    Be sure to check out the :doc:`getting started guide <../../get-started>` to learn about the common files required to execute the following recipe.
+    For example, you'll need to declare the variables for ``project``, ``api_token``.
+    These global variables that allow terraform to access the Aiven console and instruct deployments, are normally configured in a separate file called ``secret.tfvars`` and called 
+    in the terraform command line using the option ``--var-file=<path/to/tfvars file>``.
+
+The content of the ``secret.tfvars`` file, looks like this:
+
+.. code::
+
+  project = "my-aiven-project"
+  aiven_api_token = "<AIVEN_TOKEN>"
+  prefix = "anytext"
+
+The ``service.tf`` file for the provisioning af these 3 services and integrations is this:
+
+.. code::
+
+    terraform {
+      required_providers {
+        aiven = {
+          source  = "aiven/aiven"
+          version = ">=3.2.1"
+    #      version = ">= 2.0.0, < 3.0.0"
+        }
+      }
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_api_token
+    }
+
+    data "aiven_project" "prj" {
+      project = var.project
+    }
+
+    resource "aiven_pg" "demo" {
+      project      = var.project
+      service_name = "demo-postgres"
+      cloud_name   = "google-europe-north1"
+      plan         = "business-4"
+    }
+
+    resource "aiven_kafka" "kf" {
+      project                 = var.project
+      cloud_name              = "azure-norway-west"
+      plan                    = "startup-2"
+    #  service_name            = "${var.prefix}kf"
+      service_name            = "kf"
+      maintenance_window_dow  = "saturday"
+      maintenance_window_time = "10:00:00"
+      kafka_user_config {
+        kafka_rest      = true
+        kafka_connect   = false
```

Comment:
I'm checking with Aiven Terraform Provider team on default value for this flag. In case it's ``false``, we don't need to explicitly set it.

Reply:
sometimes it adds to readability when properties are added even if they have default values. 
Just because our recies are also a way to show how things work not just execute them blindly. 
My personal experience is that having more properties in these examples help with understanding how things work.